desc/objectives/_free_boundary.py

```diff
@@ -354,10 +354,13 @@ class BoundaryError(_Objective):
         Equilibrium that will be optimized to satisfy the Objective.
     field : MagneticField
         External field produced by coils.
-    s : int
+    st : int
```

Comment:
I think we could leave this as just `s` but allow it to be a tuple of two ints. So if its just a single int, we use the same for both, otherwise we treat it as `st` and `sz`